cl/donate/models.py

```diff
@@ -164,3 +166,85 @@ def __str__(self) -> str:
             self.monthly_donation_amount,
             self.get_payment_provider_display(),
         )
+
+
+class NeonWebhookEvents(models.Model):
```

Comment:
Shouldn't this be singular? E.g.:
```suggestion
class NeonWebhookEvent(models.Model):
```

Reply:
You're right! I'll rename the class.